.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+.eslintcache
```

Comment:
It would be a good idea to check how many of the files listed in here are actually present in your project. If they are not there (such as the coverage folder, `.env` files etc), they can be removed from this file